drivers/enc28j60/enc28j60.c

```diff
@@ -450,13 +447,38 @@ static int nd_init(netdev_t *netdev)
     return 0;
 }
 
+/**
+ * PKTIF does not reliably report the status of pending packets.
+ * Checking EPKTCNT is the suggested workaround.
+ * Returns the number of pending packets.
+ */
+static int rx_interrupt(netdev_t *netdev)
+{
+    enc28j60_t *dev = (enc28j60_t *)netdev;
+    int pkg_cnt = cmd_rcr(dev, REG_B1_EPKTCNT , 1);
```

Comment:
```suggestion
    int pkg_cnt = cmd_rcr(dev, REG_B1_EPKTCNT, 1);
```

Reply:
@benpicco may I force push the change?

Reply:
sure!

Reply:
@benpicco sorry, didn't got `uncrustify` up and running yet.

Reply:
That's why we have CI :smiley: